addon/nodes/paragraphWithConfig.ts

```diff
@@ -50,7 +51,7 @@ export const paragraphWithConfig: (
         default: DEFAULT_ALIGNMENT,
       },
       indentationLevel: {
-        default: 0,
+        default: null,
```

Comment:
What is the reason to change the default to `null`?

Reply:
The `toDOM` checks whether `indentationLevel` is an integer and adds an attribute if it is, so if we default to 0, it adds a `data-indentation-level=0` attribute to every paragraph, unless we specifically set `indentationLevel` to be null. This is part of minimising changes to the DOM that have no practical input, so reducing the changes if they're inside an element with a `rdf:XMLLiteral` datatype.